docker-compose.yaml.mako

```diff
@@ -22,9 +22,12 @@ build:
     - USER_ID
     - GROUP_ID
     - CI
+    - PGUSER=www-data
+    - PGPASSWORD=www-data
+    - PGDATABASE=geomapfish_tests
   stdin_open: true
   tty: true
-  command: ${'$'}{RUN}
+  command: ["wait-for-db", "${'$'}{RUN}"]
```

Comment:
Why adding the quotes? For me it's more readable without the quotes ...

Reply:
Because this is the prefered form.